lib/build.js

```diff
@@ -338,6 +343,11 @@ export default {
     targetName = targetName || this.activeTarget[p];
 
     Promise.resolve(this.targets[p]).then(targets => {
+      if (atom.packages.isPackageActive('linter')) {
```

Comment:
Of course, `linter` may not be the only package providing the linterRegistry. Just the most likely...